src/main/scala/com/gu/i18n/CountryGroup.scala

```diff
@@ -301,7 +301,10 @@ object CountryGroup {
     locale.getISO3Country.toUpperCase -> country
   }.toMap
 
-  def countryByCode(str: String): Option[Country] = countriesByISO2.get(str) orElse countriesByISO3.get(str)
+  def countryByCode(str: String): Option[Country] = {
+    val code = str.toUpperCase
+    countriesByISO2.get(code) orElse countriesByISO3.get(code)
```

Comment:
are the codes in countries byISO2 uppercase ? you could do c.alpha2.toUpperCase in line 297 if not

Reply:
Yeah, they're all uppercase.